elegant/gui/pose_annotation.py

```diff
@@ -213,6 +250,18 @@ def _set_widths_with_notification(self, width_tck):
     def set_widths_to_default(self):
         self._set_widths_with_notification(self.get_default_widths())
 
+    def _pca_smooth_widths(self, width_tck, mean_widths):
+        basis_shape = self.width_pca_basis.shape[1]
```

Comment:
Good idea to refactor out the logic of PCA smoothing into a helper function that does everything except the notification. Except you forgot to then make `pca_smooth_widths` use this helper function. So now there's duplicated code laying around.